src/any.js

```diff
@@ -0,0 +1,30 @@
+import is from './is';
+
+/**
+ * Check if any element verify a function
+ * @param {*} value the array / object to check
+ * @param {*} criteria the criteria
+ */
+export function any(value, criteria) {
```

Comment:
Please change to more significative identifiers 
I suggest `enumerable`, `matcher`